README.md

```diff
@@ -22,26 +41,183 @@ allprojects {
 }
 ```
 
-2. Add your dependency to your app's `build.gradle` file:
+### Available Artifiacts
+
+2. Add relevant dependencies to your app's `build.gradle` file:
 
 ```groovy
-implementation 'io.customer.android:sdk:<version-here>'
+implementation 'io.customer.android:tracking:<version-here>'
+implementation 'io.customer.android:messaging-push-fcm:<version-here>'
 ```
 
-Replace `version-here` with the the latest version: ![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.customer.android/sdk/badge.svg)
+Replace `version-here` with the the latest version: ![Maven Central](https://maven-badges.herokuapp.com/maven-central/io.customer.android/tracking/badge.svg)
 
-# Documentation
+# Initialize the SDK
 
-...Not yet ready. Under development. 
+Before you can use the Customer.io SDK, you need to initialize it. `CustomerIO` is a singleton: you'll create it once and re-use it across your application.
 
-# Contributing
+```kotlin
+class App : Application() {
+   override fun onCreate() {
+       super.onCreate()
+      val customerIO = CustomerIO.Builder(
+            siteId = "your-site-id",
+            apiKey = "your-api-key",
+            appContext = this
+        ).build()
+   }
+}
+```
+The Builder for CustomerIO exposes configuration options for features such `region`,
+`timeout`. 
+
+> A best practice is to initialize CustomerIO in the Application class, this way you will be able to access that instance from any part of your application using the instance() method:
```

Comment:
> using the instance() method:

I see a `:` suggesting that maybe there is supposed to be more written after this?